Documentation/CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md

```diff
@@ -50,12 +50,20 @@ If any setting is unspecified, a suitable default will be used automatically.
 
 - `blockPoolName`: The metadata name of the CephBlockPool CR where the rados namespace will be created.
 
+- `mirroring`: Sets up mirroring of the rados namespace (requires Ceph v20 or newer)
+    - `enabled`: whether mirroring is enabled on that rados namespace (default: false)
+    - `mode`: mirroring mode to run, possible values are "pool" or "image" (required). Refer to the [mirroring modes Ceph documentation](https://docs.ceph.com/docs/master/rbd/rbd-mirroring/#enable-mirroring) for more details
+    - `remoteNamespace`: Name of the rados namespace on the peer cluster where the namespace should get mirrored.
+    - `snapshotSchedules`: schedule(s) snapshot at the **rados namespace** level. One or more schedules are supported.
+        - `interval`: frequency of the snapshots. The interval can be specified in days, hours, or minutes using d, h, m suffix respectively.
+        - `startTime`: optional, determines at what time the snapshot process starts, specified using the ISO 8601 time format.
```

Comment:
Two things:

1. An example interval and start time would be helpful for users.
2. I don't see any note about required/optional on 3 of these fields, where that is noted above.

Reply:
>An example interval and start time would be helpful for users.

I have explained in the example, under `### Mirroring`

>I don't see any note about required/optional on 3 of these fields, where that is noted above.
All fields are optional, not sure if we do that with any where else